cmd/testing/cli/harness.go

```diff
@@ -102,3 +105,35 @@ func TearDown(testDir string, tee *Tee) {
 		}
 	}
 }
+
+// PrepareStdinResponse creates a temproary file with a prepared content which then is used as os.Stdin to test user input.
+// The orignal os.Stdin file handle is returned as well as the path of the file containing the canned stdin responses.
+func PrepareStdinResponse(s string, t *testing.T) (*os.File, string) {
```

Comment:
This is not used in this commit. But overall I can see delete_test.go uses PrepareStdinResponse() in another commit. Is this fine or we should keep them together?

Reply:
That's fine. It's a helper really. I should have staged in hunks. But in this case it really does not matter.